explos/channel-manager/channels.go

```diff
@@ -0,0 +1,205 @@
+package main
+
+import (
+	"berty.tech/berty/v2/go/pkg/bertybot"
```

Comment:
can you run automatic golang formatter, i.e., `goimports` or better `gofumpt`?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inefficient